src/main/java/com/marginallyclever/makelangelo/preview/PreviewPanel.java

```diff
@@ -221,16 +240,35 @@ public void display(GLAutoDrawable glautodrawable) {
 		
 		paintBackground(gl2);
 		paintCamera(gl2);
-		
-		gl2.glLineWidth((float)camera.getZoom());
 
 		for( PreviewListener p : previewListeners ) {
 			gl2.glPushMatrix();
 			p.render(gl2);
 			gl2.glPopMatrix();
 		}
+
+		//paintCursor(gl2);
```

Comment:
please remove commented code

Reply:
added a comment to explain what it is for.  future me will be glad it is there.